src/services/request.service.js

```diff
@@ -12,12 +12,13 @@ export default class RequestService {
    * @returns {Object} Saved trip request details
     */
   static handleTripRequest(tripRequestData) {
+    console.log(tripRequestData);
     return request.create(
       tripRequestData,
       {
         fields: [
           'userId',
-          'travelTo',
+          'destination',
```

Comment:
Check comment above in migrations/models

Reply:
I change this one